client/app/components/composites/ListingCard/ListingCard.css

```diff
@@ -105,6 +105,13 @@
   padding-top: 5px;
 }
 
+.orderTypeLabel {
+  flex-basis: 68%;
+  font-size: var(--ListingCard_fontSizePrice);
+  font-weight: var(--ListingCard_fontWeightMedium);
+  line-height: 1em;
+}
+
```

Comment:
The styling appears to be ~same as in priceWrapper + price, so maybe these could be combined somehow? As they fill the same function, I think they could both be inside the `.priceWrapper` div to make the structure more obvious